corehq/ex-submodules/casexml/apps/phone/restore.py

```diff
@@ -436,18 +436,13 @@ def finish_sync(self):
 
     def _new_sync_log(self):
         previous_log_id = None if self.is_initial else self.last_sync_log._id
-        previous_log_rev = None if self.is_initial else self.last_sync_log._rev
-        last_seq = str(get_db().info()["update_seq"])
         new_synclog = SimplifiedSyncLog(
-            _id=SyncLog.get_db().server.next_uuid(),
```

Comment:
shouldn't this still be set to a UUID?

Reply:
It will automatically get set in the `save_synclog_to_sql` method, if it is set to None.